.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+updates:
```

Comment:
Feel free to add `Go` too, just in case 😉

Reply:
Done